flink-core/src/main/java/org/apache/flink/core/fs/ConnectionLimitingFactory.java

```diff
@@ -62,7 +62,9 @@ public FileSystem create(URI fsUri) throws IOException {
 		FileSystem original = factory.create(fsUri);
 		return new LimitedConnectionsFileSystem(original,
 				settings.limitTotal, settings.limitOutput, settings.limitInput,
-				settings.streamOpenTimeout, settings.streamInactivityTimeout);
+				settings.streamOpenTimeout, settings.streamInactivityTimeout,
```

Comment:
please reformat previous arguments (one per line) since you have just touched this one